From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 29 Oct 2018 01:56:50 +0000
Subject: slub: Avoid ABI change in 4.9.131
Forwarded: not-needed

Commit b052d04aa3fe "slub: make ->cpu_partial unsigned int" changed
the type of kmem_cache::cpu_partial from int to unsigned int.  This
is a private member and the structure layout hasn't changed, so it's
not a real ABI change.  Hide it from genksyms.

---
--- a/include/linux/slub_def.h
+++ b/include/linux/slub_def.h
@@ -68,7 +68,11 @@ struct kmem_cache {
 	int object_size;	/* The size of an object without meta data */
 	int offset;		/* Free pointer offset. */
 	/* Number of per cpu partial objects to keep around */
+#ifndef __GENKSYMS__
 	unsigned int cpu_partial;
+#else
+	int cpu_partial;
+#endif
 	struct kmem_cache_order_objects oo;
 
 	/* Allocation and freeing of slabs */
